### PR TITLE
docs: fix environments.rst

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- remove environment-specific configuration files 

It seems there is no such a thing in CI4.

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/general/environments.rst [hunk 1]
```diff
@@ -97,17 +97,9 @@ is affected.
 Error Reporting
 ---------------
 
-Setting the ENVIRONMENT constant to a value of 'development' will cause
+Setting the ENVIRONMENT constant to a value of ``development`` will cause
 all PHP errors to be rendered to the browser when they occur.
 Conversely, setting the constant to 'production' will disable all error
```
Comment (MGatner) [inline user_guide_src/source/general/environments.rst:102]:
```suggestion
Conversely, setting the constant to ``production`` will disable all error
```

File: user_guide_src/source/general/environments.rst [hunk 2]
```diff
@@ -97,17 +97,9 @@ is affected.
...
 :doc:`good security practice </concepts/security>`.
 
-Configuration Files
--------------------
-
-Optionally, you can have CodeIgniter load environment-specific
-configuration files. This may be useful for managing things like
-differing API keys across multiple environments. This is described in
-more detail in the Handling Different Environments section of the
-:doc:`Working with Configuration Files </general/configuration>` documentation.
```
Comment (MGatner) [inline user_guide_src/source/general/environments.rst:113]: Why was this removed? Does that section not exist?

**Review by MGatner — CHANGES_REQUESTED** [review]

**Comment by MGatner** [comment]
I see now, we don't have such a thing as "Environment-specific configuration files". Boot files are the closest thing, and those are already covered.

I agree with this removal.

**Comment by kenjis** [comment]
Yes, CI3 has environment-specific configuration files, but CI4 does not have.

**Review by paulbalandan — APPROVED** [review]

File: user_guide_src/source/general/environments.rst (hunk 1 above)
Comment (kenjis) [reply user_guide_src/source/general/environments.rst:102]: Fixed.

**Pull request merged**